test/utils/fixtures.ts

```diff
@@ -133,6 +135,14 @@ export const QUOTE_REQUEST_CLASSIC_FE_SEND_PORTION = makeClassicRequest({
   sendPortionEnabled: true,
   portion: FLAT_PORTION,
 });
+export const QUOTE_REQUEST_CLASSIC_FE_ENABLE_FEE_ON_TRANSFER = makeClassicRequest({
+  configs: [
+    {
+      routingType: RoutingType.CLASSIC,
+      enableFeeOnTransferFeeFetching: true,
```

Comment:
this would override other set keys like `protocols`

Reply:
Good catch..